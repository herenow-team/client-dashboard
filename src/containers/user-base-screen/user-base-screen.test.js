import React from 'react'
import renderer from 'react-test-renderer'
import {BrowserRouter as Router} from 'react-router-dom'
import UserBaseScreen from '.'

describe('<BaseScreen /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <UserBaseScreen>
            <div>Some content</div>
          </UserBaseScreen>
        </Router>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
