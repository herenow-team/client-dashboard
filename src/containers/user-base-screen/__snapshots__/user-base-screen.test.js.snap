// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseScreen /> component should render correctly 1`] = `
<div>
  <header
    className="header"
  >
    <a
      className="has-text-success logo"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="herenow"
        className="logo"
        src="nothing"
      />
    </a>
  </header>
  <div
    className="container"
  >
    <div>
      Some content
    </div>
  </div>
</div>
`;
